src/components/CityList/index.jsx

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from "react";
+import { fetchCitiesData } from "../ApiData/data";
+import { Link } from "react-router-dom";
+import "animate.css/animate.min.css";
+import "./styled.scss";
+
+function CityList() {
+  const [cities, setCities] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const data = await fetchCitiesData();
+        setCities(data);
+      } catch (error) {
+        console.error("Ошибка при загрузке данных", error);
+      }
+    };
+
+    fetchData();
+  }, []);
+
+  return (
+    <div className="animate__animated animate__fadeIn weather_table">
+      <ul>
+        {cities.map((city) => (
```

Comment:
Please use the [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).
```suggestion
        {cities.map(({main: { temp, humidity, speed: windSpeed, pressure }}) => (
```

Reply:
{cities.map((city) => {
          const { id, name, main, wind } = city;
          const { temp, humidity, pressure } = main;
          const { speed: windSpeed } = wind;

          How about this?

Reply:
Yep, it looks good and clear!